stage0/layout.ld

```diff
@@ -35,6 +35,7 @@ HIDDEN(SEGMENT_PRESENT = 1 << (32 + 15)); /* P */
 HIDDEN(SEGMENT_USER = 1 << (32 + 12)); /* S */
 HIDDEN(SEGMENT_CODE = 1 << (32 + 11));
 HIDDEN(SEGMENT_WRITABLE = 1 << (32 + 9));
+HIDDEN(SEGMENT_ACCESSED = 1 << (32 + 8));
```

Comment:
minor nit, but can you move both `SEGMENT_WRITABLE` and `SEGMENT_ACCESSED` below `SEGMENT_CONFORMING`? This would order the bits nicely.

Reply:
Good idea, done.